frontend/src/design/components/DeleteObjectWithFrictionModal.js

```diff
@@ -95,6 +95,7 @@ export const DeleteObjectWithFrictionModal = (props) => {
               variant="contained"
               onClick={() => {
                 deleteFunction(deleteFromAWS);
+                setConfirmValue(null);
```

Comment:
I am not sure why this is needed; so it seems to be preventing that the next time the delete confirmation modal opens the confirm value is the target, but doesn't it get the null value from `const [confirmValue, setConfirmValue] = useState(null);` when it gets re-rendered?

Reply:
I imagine the modal hides but is not re-rendered if you removed 2 groups in succession - so confirmValue never reset anywhere previously